src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+import { TodosFilter } from '../TodosFilter/TodoFilter';
+
+export const Footer = () => {
+  const { todos, updateTodos } = useContext(TodosContext);
+
+  const handleClearCompleted = () => {
+    updateTodos(todos.filter(todo => todo.completed !== 'completed'));
```

Comment:
The completed property of a todo item is likely a boolean so checking for !== 'completed' seems incorrect. Instead, it should be checking for !todo.completed